datasets/kor_sarcasm/kor_sarcasm.py

```diff
@@ -0,0 +1,73 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Korean Sarcasm Detection Dataset"""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+
+import datasets
+
+
+_DESCRIPTION = """\
+This is a dataset designed to detect sarcasm in Korean because it distorts the literal meaning of a sentence
+and is highly related to sentiment classification.
+"""
+
+_HOMEPAGE = "https://github.com/SpellOnYou/korean-sarcasm"
+
+_LICENSE = "MIT License"
+
+_TRAIN_DOWNLOAD_URL = "https://raw.githubusercontent.com/SpellOnYou/korean-sarcasm/master/data/jiwon/train.csv"
+_TEST_DOWNLOAD_URL = "https://raw.githubusercontent.com/SpellOnYou/korean-sarcasm/master/data/jiwon/test.csv"
+
+
+class KorSarcasm(datasets.GeneratorBasedBuilder):
+    """Korean Sarcasm Detection Dataset"""
+
+    VERSION = datasets.Version("1.1.0")
+
+    def _info(self):
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {"tokens": datasets.Value("string"), "label": datasets.features.ClassLabel(names=["0", "1"])}
```

Comment:
Could you use more explicit label class names please ?
For example
```python
names=["no-sarcasm", "sarcasm"]
```

you'll have to regenerate the dataset_infos.json after this change

Reply:
i get a `KeyError: '1'` when I try to rename the class names more explicitly. is there a way I can change this without modifying the original class labels in the dataset?